.github/workflows/merge-feature-tests-to-main.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: "Publish Feature Tests to Build"
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "feature-tests/**"
+      - ".github/workflows/manual-publish-feature-tests-to-dev.yaml"
+  workflow_dispatch:
+    inputs:
```

Comment:
inputs section can be removed as should not be able to input feature branch into build workflow

Reply:
Thanks Dom, changed made on latest push